server/apps/publish/formatters/aap_formatter_common.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import superdesk
+
+
+def map_priority(priority):
+    """
+    Map the IPTC NewsML Priority to Legacy
+    :param int priority:
+    :return: one character legacy priority
+    """
+    mapping = {
+        1: 'f',
+        2: 'u',
+        3: 'b',
+        4: 'r',
+        5: 'z',
+        6: 'd'
+    }
```

Comment:
Would it be easier to use an `Enum` for the list of valid Priority values? (there exists a backport to Python3.3)
Just an idea, in case it can somehow simplify the code that relies on this...